cpp/libsarmata-client/sarmata_client.cpp

```diff
@@ -41,7 +41,7 @@ DefineGrammarResponse SarmataClient::DefineGrammar(const SarmataSessionConfig& c
 }
 
 
-std::vector<RecognizeResponse> SarmataClient::Recognize(const SarmataSessionConfig& config, unsigned int audio_sample_rate_hz, const std::string& audio_byte_content) const {
+std::vector<RecognizeResponse> SarmataClient::Recognize(const SarmataSessionConfig& config, const std::string& audio_byte_content) const {
```

Comment:
Could you consider passing `audio_sample_rate_hz` as an explicit parameter and putting it into `SarmataSessionConfig` inside this method? Hidding this parameter among many others in `SarmataSessionConfig` is potential place error-prone place. Is this value validated on the server side?